src/tests/test_process.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+import subprocess
+import os
+import signal
+import tempfile
+from time import sleep
+from datetime import datetime
```

Comment:
Import should be in a order.

Reply:
prooflink, please

Reply:
Imports should be in alphabetic order after being grouped. It's the flake8 H306 rule, even though it's not a part of PEP8 as far as I understand.

Reply:
It's rather esoteric rule because I've never seen it's application while have flake8 checks enabled in emacs python-mode.

I've followed it but only for newly created modules.